beetmoverscript/task.py

```diff
@@ -17,6 +19,27 @@ def validate_task_schema(context):
     scriptworker.client.validate_json_schema(context.task, task_schema)
 
 
+def validate_task_scopes(context, manifest):
+    # make sure scopes exist and are properly set
+    scopes = context.task['scopes']
+    for scope in scopes:
+        if scope.startswith("project:releng:beetmover:"):
+            signing_cert_name = scope.split(':')[-1]
+            if re.search('^[0-9A-Za-z_-]+$', signing_cert_name) is not None:
+                break
+            log.warning('scope {} is malformed, skipping!'.format(scope))
+    else:
+        log.critical("No beetmover scopes!")
+        sys.exit(3)
+
+    # prevent uncoordination between the bucket supposed to use for beetmove
+    # and the current credentials scopes
+    if (signing_cert_name not in manifest['s3_prefix_dated'] or
+            signing_cert_name not in manifest['s3_prefix_latest']):
```

Comment:
I am not sure if the existing check was to be replaced by this or just enriched. For safety reasons I left it too. I am thinking that any scopes munging to access the nightly/release buckets will be prevented by your suggested check. The existing one should catch munging scopes across various low-security trees trying to access each other buckets. Not sure if it makes sense. Happy to remove it if it doesn't. 

I'll go ahead and land this for now and deployed it on workers. Will need to make changes to address the checksums anyway so we'll most likely follow-up with future bumps anyway.